### PR TITLE
Daily Standup View Screen 2

### PR DESCRIPTION
**Pull request by Chittauri**

### CODE REVIEW

File: common/src/main/java/com/publicissapient/kpidashboard/common/model/application/FieldMapping.java [hunk 1]
```diff
@@ -462,4 +471,13 @@ public void setJiraIssueTypeNames(String[] jiraIssueTypeNames) {
 	public boolean getNotificationEnabler() {
 		return notificationEnabler;
 	}
+
+	public List<String> getJiraStatusForInProgressKPI154() {
+		return jiraStatusForInProgressKPI119;
```
Comment (shunaray) [inline common/src/main/java/com/publicissapient/kpidashboard/common/model/application/FieldMapping.java:476]: getJiraStatusForInProgressKPI154 imethod name is not matching return fieldname

File: common/src/main/java/com/publicissapient/kpidashboard/common/util/DateUtil.java [hunk 2]
```diff
@@ -199,6 +198,14 @@ public static boolean isWithinDateTimeRange(LocalDateTime targetDate, LocalDateT
 		return !targetDate.isBefore(startDate) && !targetDate.isAfter(endDate);
 	}
 
+	public static boolean equalAndAfterTime(LocalDateTime targetDate, LocalDateTime startDate) {
+		return targetDate.isAfter(startDate) || targetDate.isEqual(startDate);
+	}
+
+	public static boolean equalAndBeforTime(LocalDateTime targetDate, LocalDateTime startDate) {
+		return targetDate.isBefore(startDate) || targetDate.isEqual(startDate);
```
Comment (shunaray) [inline common/src/main/java/com/publicissapient/kpidashboard/common/util/DateUtil.java:206]: can be simplified return !targetDate.isAfter(startDate);

File: customapi/src/main/java/com/publicissapient/kpidashboard/apis/util/CommonUtils.java [hunk 3]
```diff
@@ -502,4 +503,41 @@ public static String convertIntoDays(Integer minutes) {
...
 		return returnString.toString();
 	}
+
+	/**
+	 * Method to get next working date i.e excluding sat sun
+	 * @param currentDate currentDate
+	 * @param daysToAdd count of days to add
+	 * @return
+	 */
+	public static java.time.LocalDate getNextWorkingDate(java.time.LocalDate currentDate, long daysToAdd) {
```
Comment (shunaray) [inline customapi/src/main/java/com/publicissapient/kpidashboard/apis/util/CommonUtils.java:513]: here after adding countOfDays we don't want weekend right?
else we can use getWorkingDayAfterAdditionofDays method

File: customapi/src/main/java/com/publicissapient/kpidashboard/apis/util/CommonUtils.java [hunk 4]
```diff
@@ -502,4 +503,41 @@ public static String convertIntoDays(Integer minutes) {
...
+		java.time.LocalDate resultDate = currentDate.plusDays(daysToAdd);
+
+		while (resultDate.getDayOfWeek() == DayOfWeek.SATURDAY || resultDate.getDayOfWeek() == DayOfWeek.SUNDAY) {
+			resultDate = resultDate.plusDays(1);
+		}
+
+		return resultDate;
+	}
+
+	public static String convertSecondsToDays(int inputSeconds) {
```
Comment (shunaray) [inline customapi/src/main/java/com/publicissapient/kpidashboard/apis/util/CommonUtils.java:523]: java doc

File: UI/src/app/dashboard/daily-scrum/daily-scrum.component.css [hunk 5]
```diff
@@ -1,64 +1,191 @@
...
     font-weight: bold;
+    display: inline-block;
+    font-size: 14px;
+    color: #000;
+    padding: 0;
+    align-items: center;
+    margin-left: 4px;
+    border: 2px solid #000;
+    text-align: center;
+    font-family: Arial;
```
[the remark this answers is not in the document]
Comment (risshukl0) [reply UI/src/app/dashboard/daily-scrum/daily-scrum.component.css:94]: font-family Arial is intentional here. Default fonts don't look good inside the circular btn

**Review by HinPublicis — APPROVED** [review]

**Review by risshukl0 — APPROVED** [review]

**Comment by hirbabar** [comment]
Nice

**Review by hirbabar — APPROVED** [review]
Good work

**Review by shunaray — APPROVED** [review]

**Pull request merged**